Translations update from Fedora Weblate

weblate opened the pull request:
Translations update from [Fedora Weblate](https://translate.fedoraproject.org) for [rpminspect/main](https://translate.fedoraproject.org/projects/rpminspect/main/).



Current translation status:

![Weblate translation status](https://translate.fedoraproject.org/widgets/rpminspect/-/main/horizontal-auto.svg)

dcantrell committed:
Added translation using Weblate (Chinese (Hong Kong) (zh_HK))

dcantrell committed:
Added translation using Weblate (Spanish (Latin America))

dcantrell committed:
Added translation using Weblate (Chinese (Traditional) (zh_TW))

dcantrell committed:
Added translation using Weblate (Chinese (Simplified) (zh_CN))

dcantrell committed:
Added translation using Weblate (Pirate (pr))

dcantrell committed:
Added translation using Weblate (Klingon)

dcantrell committed:
Added translation using Weblate (Klingon (pIqaD))

dcantrell committed:
Added translation using Weblate (French (Antigua and Barbuda))

dcantrell committed:
Added translation using Weblate (English (Pirate))

dcantrell committed:
Added translation using Weblate (Eskimo (Pacific Coast Alaskan))

dcantrell committed:
Added translation using Weblate (Amharic)

dcantrell committed:
Added translation using Weblate (Afrikaans)

dcantrell committed:
Added translation using Weblate (Arabic)

dcantrell committed:
Added translation using Weblate (Belarusian)

dcantrell committed:
Added translation using Weblate (Assamese)

dcantrell committed:
Added translation using Weblate (Asturian)

dcantrell committed:
Added translation using Weblate (Bulgarian)

dcantrell committed:
Added translation using Weblate (Bengali)

dcantrell committed:
Added translation using Weblate (Bengali (Bangladesh))

dcantrell committed:
Added translation using Weblate (Bengali (India))

dcantrell committed:
Added translation using Weblate (Catalan)

dcantrell committed:
Added translation using Weblate (Czech)

dcantrell committed:
Added translation using Weblate (Danish)

dcantrell committed:
Added translation using Weblate (German)

dcantrell committed:
Added translation using Weblate (Welsh)

dcantrell committed:
Added translation using Weblate (German (formal))

dcantrell committed:
Added translation using Weblate (German (informal))

dcantrell committed:
Added translation using Weblate (Greek)

dcantrell committed:
Added translation using Weblate (German (Luxembourg))

dcantrell committed:
Added translation using Weblate (German (Switzerland))

dcantrell committed:
Added translation using Weblate (German (Austria))

dcantrell committed:
Added translation using Weblate (English (Australia))

dcantrell committed:
Added translation using Weblate (English (United Kingdom))

dcantrell committed:
Added translation using Weblate (English (Canada))

dcantrell committed:
Added translation using Weblate (English (New Zealand))

dcantrell committed:
Added translation using Weblate (English (India))

dcantrell committed:
Added translation using Weblate (English (Ireland))

dcantrell committed:
Added translation using Weblate (Esperanto)

dcantrell committed:
Added translation using Weblate (English (Philippines))

dcantrell committed:
Added translation using Weblate (English (South Africa))

dcantrell committed:
Added translation using Weblate (Spanish (informal))

dcantrell committed:
Added translation using Weblate (Spanish (formal))

dcantrell committed:
Added translation using Weblate (Spanish)

dcantrell committed:
Added translation using Weblate (Spanish (Argentina))

dcantrell committed:
Added translation using Weblate (Spanish (Bolivia))

dcantrell committed:
Added translation using Weblate (Spanish (Chile))

dcantrell committed:
Added translation using Weblate (Spanish (Costa Rica))

dcantrell committed:
Added translation using Weblate (Spanish (Colombia))

dcantrell committed:
Added translation using Weblate (Spanish (Dominican Republic))

dcantrell committed:
Added translation using Weblate (Spanish (Mexico))

dcantrell committed:
Added translation using Weblate (Spanish (Ecuador))

dcantrell committed:
Added translation using Weblate (Spanish (Nicaragua))

dcantrell committed:
Added translation using Weblate (Spanish (Peru))

dcantrell committed:
Added translation using Weblate (Spanish (Panama))

dcantrell committed:
Added translation using Weblate (Spanish (Puerto Rico))

dcantrell committed:
Added translation using Weblate (Spanish (El Salvador))

dcantrell committed:
Added translation using Weblate (Spanish (American))

dcantrell committed:
Added translation using Weblate (Spanish (Uruguay))

dcantrell committed:
Added translation using Weblate (Basque)

dcantrell committed:
Added translation using Weblate (Spanish (Venezuela))

dcantrell committed:
Added translation using Weblate (Estonian)

dcantrell committed:
Added translation using Weblate (Finnish)

dcantrell committed:
Added translation using Weblate (Persian)

dcantrell committed:
Added translation using Weblate (French)

dcantrell committed:
Added translation using Weblate (Filipino)

dcantrell committed:
Added translation using Weblate (French (formal))

dcantrell committed:
Added translation using Weblate (French (Belgium))

dcantrell committed:
Added translation using Weblate (French (informal))

dcantrell committed:
Added translation using Weblate (French (Canada))

dcantrell committed:
Added translation using Weblate (French (Senegal))

dcantrell committed:
Added translation using Weblate (French (Luxembourg))

dcantrell committed:
Added translation using Weblate (French (Louisiana))

dcantrell committed:
Added translation using Weblate (French (Switzerland))

dcantrell committed:
Added translation using Weblate (Frisian (Northern))

dcantrell committed:
Added translation using Weblate (Frisian (Eastern))

dcantrell committed:
Added translation using Weblate (Galician)

dcantrell committed:
Added translation using Weblate (Frisian)

dcantrell committed:
Added translation using Weblate (Irish)

dcantrell committed:
Added translation using Weblate (Hindi)

dcantrell committed:
Added translation using Weblate (Hawaiian)

dcantrell committed:
Added translation using Weblate (Hebrew)

dcantrell committed:
Added translation using Weblate (Gujarati)

dcantrell committed:
Added translation using Weblate (Hungarian)

dcantrell committed:
Added translation using Weblate (Croatian)

dcantrell committed:
Added translation using Weblate (Interlingua)

dcantrell committed:
Added translation using Weblate (Icelandic)

dcantrell committed:
Added translation using Weblate (Indonesian)

dcantrell committed:
Added translation using Weblate (Italian)

dcantrell committed:
Added translation using Weblate (Japanese)

dcantrell committed:
Added translation using Weblate (Georgian)

dcantrell committed:
Added translation using Weblate (Italian (Switzerland))

dcantrell committed:
Added translation using Weblate (Kazakh)

dcantrell committed:
Added translation using Weblate (Khmer (Central))

dcantrell committed:
Added translation using Weblate (Kannada)

dcantrell committed:
Added translation using Weblate (Lithuanian)

dcantrell committed:
Added translation using Weblate (Korean)

dcantrell committed:
Added translation using Weblate (Latvian)

dcantrell committed:
Added translation using Weblate (Burmese)

dcantrell committed:
Added translation using Weblate (Marathi)

dcantrell committed:
Added translation using Weblate (Malayalam)

dcantrell committed:
Added translation using Weblate (Norwegian Bokmål)

dcantrell committed:
Added translation using Weblate (Dutch)

dcantrell committed:
Added translation using Weblate (Norwegian Nynorsk)

dcantrell committed:
Added translation using Weblate (Odia)

dcantrell committed:
Added translation using Weblate (Occitan)

dcantrell committed:
Added translation using Weblate (Polish)

dcantrell committed:
Added translation using Weblate (Punjabi)

dcantrell committed:
Added translation using Weblate (Portuguese)

dcantrell committed:
Added translation using Weblate (Portuguese (formal))

dcantrell committed:
Added translation using Weblate (Portuguese (Angola))

dcantrell committed:
Added translation using Weblate (Portuguese (informal))

dcantrell committed:
Added translation using Weblate (Portuguese (Brazil, informal))

dcantrell committed:
Added translation using Weblate (Portuguese (Brazil))

dcantrell committed:
Added translation using Weblate (Portuguese (Brazil, formal))

dcantrell committed:
Added translation using Weblate (Romanian)

dcantrell committed:
Added translation using Weblate (Russian (formal))

dcantrell committed:
Added translation using Weblate (Portuguese (Portugal))

dcantrell committed:
Added translation using Weblate (Russian (Ukraine))

dcantrell committed:
Added translation using Weblate (Sinhala)

dcantrell committed:
Added translation using Weblate (Russian (informal))

dcantrell committed:
Added translation using Weblate (Albanian)

dcantrell committed:
Added translation using Weblate (Serbian)

dcantrell committed:
Added translation using Weblate (Slovak)

dcantrell committed:
Added translation using Weblate (Telugu)

dcantrell committed:
Added translation using Weblate (Tamil)

dcantrell committed:
Added translation using Weblate (Swedish)

dcantrell committed:
Added translation using Weblate (Thai)

dcantrell committed:
Added translation using Weblate (Turkish)

dcantrell committed:
Added translation using Weblate (Urdu)

dcantrell committed:
Added translation using Weblate (Vietnamese)

dcantrell committed:
Added translation using Weblate (Ukrainian)

dcantrell committed:
Added translation using Weblate (German (Walser))

NorwayFun committed:
Translated using Weblate (Georgian)

Currently translated at 3.0% (22 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ka/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 3.7% (27 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

NorwayFun committed:
Translated using Weblate (Georgian)

Currently translated at 3.2% (23 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ka/

git-pear committed:
Translated using Weblate (Czech)

Currently translated at 0.4% (3 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/cs/

oersen committed:
Translated using Weblate (Turkish)

Currently translated at 3.6% (26 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/tr/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 6.3% (45 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 1.8% (13 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 7.9% (57 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 8.2% (59 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 10.3% (74 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 2.9% (21 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 10.2% (73 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 17.2% (123 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 12.6% (90 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

copperii committed:
Translated using Weblate (Finnish)

Currently translated at 3.2% (23 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/fi/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 28.6% (204 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 32.6% (233 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 41.0% (293 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 52.3% (373 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 60.7% (433 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 64.9% (463 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 73.3% (523 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

hopparz committed:
Translated using Weblate (Hungarian)

Currently translated at 0.7% (5 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/hu/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 91.5% (653 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (713 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (713 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

NorwayFun committed:
Translated using Weblate (Georgian)

Currently translated at 3.3% (24 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ka/

NorwayFun committed:
Translated using Weblate (Georgian)

Currently translated at 4.4% (32 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ka/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 12.7% (91 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

goeranu committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (713 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/sv/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 15.9% (114 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

rffontenelle commented:
@dcantrell Sorry for pinging you, but this PR is open with translations contributions for 6 months now. Could you please consider merging this?

Commit:
Translated using Weblate (Bengali (Bangladesh))

Currently translated at 0.2% (2 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/bn_BD/

Commit:
Translated using Weblate (Korean)

Currently translated at 16.1% (115 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 16.2% (116 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 16.2% (116 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 16.4% (117 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 16.4% (117 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 16.5% (118 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 16.5% (118 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 16.6% (119 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 16.6% (119 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 16.6% (119 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Korean)

Currently translated at 16.8% (120 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 16.8% (120 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

Commit:
Translated using Weblate (Georgian)

Currently translated at 4.6% (33 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ka/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 18.0% (129 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 19.2% (137 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 21.4% (153 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 21.4% (153 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 26.6% (190 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 53.5% (382 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 57.6% (411 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 19.2% (137 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 78.6% (561 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 78.6% (561 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 92.7% (661 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 92.7% (661 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 93.9% (670 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 20.0% (143 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 95.3% (680 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.5% (710 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 20.1% (144 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

rffontenelle commented:
@dcantrell Is there anything blocking merging the translations contributed to rpminspect? Please let me know if any help is needed on Weblate side.

simmon-nplob committed:
Translated using Weblate (Korean)

Currently translated at 20.4% (146 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/ko/

rffontenelle committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (713 of 713 strings)

Translation: rpminspect/main
Translate-URL: https://translate.fedoraproject.org/projects/rpminspect/main/pt_BR/

rffontenelle commented:
@dcantrell ping
